UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentView.cs

```diff
@@ -7,8 +7,34 @@ namespace PlayFab
 
     public class PlayFabMultiplayerAgentView : MonoBehaviour
     {
+        /// <summary>
+        /// Static reference to the current active instance of the PlayFabMultiplayerAgentView.
+        /// </summary>
+        public static PlayFabMultiplayerAgentView Current { get; private set; } = null;
+        
         private float _timer;
 
+        /// <summary>
+        /// Awake constructor
+        /// </summary>
+        private void Awake()
+        {
+            Debug.Log($"{Time.fixedTime} {nameof(PlayFabMultiplayerAgentView)} awake");
+            // Check if the static instance already contains a reference:
+            if(Current != null)
+            {
+                // Destroy this instance since we only ever need one PlayFabMultiplayerAgentView
+                Destroy(gameObject);
+                return;
+            }
+            else
+            {
+                // Need to keep this game object alive through scene changes.
+                DontDestroyOnLoad(this);
+                current = this;
```

Comment:
does `current` (with a lowercase 'c') exist somewhere else?

Reply:
It actually did not compile since i did not check. Was my mistake, should have been a capital C.